src/consts/models.tsx

```diff
@@ -51,6 +51,13 @@ export const ELECTRUM_VAULT_SEED_PREFIXES = {
   SEED_PREFIX_2FA_SW: '102', // Two-factor auth, using segwit
 };
 
+export interface Toast {
+  title: string;
+  description: string;
+  milisecondsAfterHide: number;
```

Comment:
did you mean "duration" here? `milisecondsAfterHide` doesn't sound self-explanatory

Reply:
changed to duration